test/OpenTelemetry.Exporter.ZPages.Tests/ZPagesExporterTests.cs

```diff
@@ -18,18 +18,21 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Linq;
+using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
 using OpenTelemetry.Exporter.ZPages.Implementation;
 using OpenTelemetry.Resources;
 using OpenTelemetry.Trace;
-
 using Xunit;
 
 namespace OpenTelemetry.Exporter.ZPages.Tests
 {
+    [Collection("ZPagesExporterTests")]
```

Comment:
I don't think you need the `Collection` here. It will default to class name. I've only ever used that when I wanted to make sure two different test classes didn't run concurrently.

Reply:
yup, thats the idea. The ZPagesExporter and ZPagesActivityTracker uses the same static variables. With that, they were running in parallel, causing some tests to fail.

Reply:
Shouldn't they have the same `Collection` name?

Currently:
```csharp
    [Collection("ZPagesActivityTrackerTests")]
    public class ZPagesActivityTrackerTests

    [Collection("ZPagesExporterTests")]
    public class ZPagesExporterTests
```

I would expect something like:
```csharp
    [Collection("ZPagesTests")]
    public class ZPagesActivityTrackerTests

    [Collection("ZPagesTests")]
    public class ZPagesExporterTests
```

Reply:
i'm removing the collections again. Let's check :)